block-producer/src/txqueue/mod.rs

```diff
@@ -22,14 +22,14 @@ mod tests;
 /// See [crate::store::ApplyBlock], that trait's `apply_block` is called when a block is sealed, and
 /// it can determine when transactions are no longer in-flight.
 #[async_trait]
-pub trait TransactionVerifier: Send + Sync + 'static {
+pub trait TransactionValidator: Send + Sync + 'static {
     /// Method to receive a `tx` for processing.
     ///
     /// This method should:
     ///
-    /// 1. Verify the transaction is valid, against the current's rollup state, and also against
+    /// Verify the transaction is valid, against the current's rollup state, and also against
     ///    in-flight transactions.
-    /// 2. Track the necessary state of the transaction until it is commited to the `store`, to
+    /// Track the necessary state of the transaction until it is commited to the `store`, to
     ///    perform the check above.
```

Comment:
We should still keep this as bullet points (even if as un-ordered list). Also, let's make sure to update the formatting.

Reply:
What do you imply by "update the formatting"?